parm/ufs/fv3/diag_table_da

```diff
@@ -2,15 +2,15 @@
 "fv3_history2d",  0,  "hours",  1,  "hours",  "time"
 "ocn_da%4yr%2mo%2dy%2hr", 1, "hours", 1, "hours", "time", 1,  "hours"
```

Comment:
@junwang-noaa @aerorahul  - This PR has included my attempt to make the ocean model outputs be instantaneous instead of time averaged.   I wanted to follow up from the conversation yesterday about what should be changed in line 3 here in addition to marking the  change from none->false in the lines below.  I've re-read the instructions at the bottom of the diag_table but it's unclear to me what else to change. Thanks in advance for the help.

Reply:
Has this been addressed yet?

Reply:
yes.  This is fine.